Sources/LiveKit/Participant/LocalParticipant.swift

```diff
@@ -339,10 +381,11 @@ public extension LocalParticipant {
                     let localTrack: LocalVideoTrack
                     let options = (captureOptions as? ScreenShareCaptureOptions) ?? room._state.roomOptions.defaultScreenShareCaptureOptions
                     if options.useBroadcastExtension {
-                        await RPSystemBroadcastPickerView.show(
-                            for: BroadcastScreenCapturer.screenSharingExtension,
-                            showsMicrophoneButton: false
-                        )
+                        guard BroadcastManager.shared.isBroadcasting else {
+                            BroadcastManager.shared.requestActivation()
+                            return nil
+                        }
+                        // Wait until broadcasting to publish track
```

Comment:
you either need to store the passed options somewhere or a log a warning that custom options should _not_ be passed here when using broadcast extension (if the passed options differ from the default options on anything besides useBroadcast:true)